src/app/Components/Pages/home/about/about.component.css

```diff
@@ -13,11 +13,13 @@
     color: white;
     font-weight: 600;
     font-size: 44px;
-    font-family: Outfit;
+    font-family: 'Outfit';
```

Comment:
Font family is same on each component please create one class for font-family and add that class